.buildkite/publish/docker-compose.yml

```diff
@@ -55,10 +56,12 @@ services:
       - postgres
       - postgres_isolated
       - mysql
+      - mysql_isolated
       - mariadb
       - mssql
       - mongo
-      - mongo-seed
+      - mongo_migrate
+      - mongo_seed
```

Comment:
Would be good to rename mongo_seed to mongo_migrate_seed maybe because it could be confusing right now

Reply:
Will do